tests/Behat/Context/Setup/OrderContext.php

```diff
@@ -54,26 +55,27 @@ public function thisOrderWithMolliePaymentIsAlreadyPaid(OrderInterface $order):
         $this->entityManager->flush();
     }
 
-    /**
-     * @param $transition
-     *
-     * @throws \SM\SMException
-     */
-    private function applyMolliePaymentTransitionOnOrder(OrderInterface $order, $transition): void
+    private function applyMolliePaymentTransitionOnOrder(OrderInterface $order, string $transition): void
     {
         foreach ($order->getPayments() as $payment) {
             /** @var PaymentMethodInterface $paymentMethod */
             $paymentMethod = $payment->getMethod();
 
-            if (true === in_array($paymentMethod->getGatewayConfig()->getFactoryName(), [MollieGatewayFactory::FACTORY_NAME, MollieSubscriptionGatewayFactory::FACTORY_NAME], true)) {
+            $gatewayConfig = $paymentMethod->getGatewayConfig();
+
+            Assert::notNull($gatewayConfig);
+
+            if (true === in_array($gatewayConfig->getFactoryName(), [MollieGatewayFactory::FACTORY_NAME, MollieSubscriptionGatewayFactory::FACTORY_NAME], true)) {
```

Comment:
I would put it into private method for better readibility

Reply:
Created separate Checker - fixed